### PR TITLE
update manual to branch v2.0.0 (55a2147b62d) 

### PR DESCRIPTION
**Pull request by whitewum**
The code is not frozen at all.

I'll have to update one more time.

### CODE REVIEW

File: docs-2.0/3.ngql-guide/3.data-types/6.list.md [hunk 1]
```diff
@@ -85,35 +83,32 @@ nebula> MATCH p = (n:player{name:"Tim Duncan"})-[:follow]->(m) \
...
-## openCypher compatibility
+## OpenCypher compatibility
 
-TODO(coding)
-<!--
-List has many incompatible with openCypher. Refer to github.
+- A composite data type (i.e., set, map, and list) **CAN NOT** be stored as properties for vertices or edges.
 
-At this time, Nebula Graph does not support using ranges inside the brackets to return ranges of the list.
+- Using ranges inside the brackets to return ranges of the list.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/3.data-types/6.list.md:98]: Use the `range()` function to return the range of a list.

File: docs-2.0/3.ngql-guide/3.data-types/8.map.md [hunk 2]
```diff
@@ -15,6 +13,8 @@ nebula> YIELD {key: 'Value', listKey: [{inner: 'Map1'}, {inner: 'Map2'}]}
 +-------------------------------------------------------------+
 ```
 
-## Map projection
+## OpenCypher compatibility
+
+- A composite data type (i.e. map, and list) cannot be stored as (vertex's or edge's) properties.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/3.data-types/8.map.md:18]: Plz keep consistent: 
A composite data type (i.e. set, map, and list) CANNOT be stored as properties of vertices or edges.

File: docs-2.0/3.ngql-guide/3.data-types/9.type-conversion.md [hunk 3]
```diff
@@ -1,37 +1,75 @@
-# Type Conversion
+# Type Conversion/Type coercions
 
-Converting an expression of a given type to another type is known as type conversion. In nGQL, type conversion is divided into explicit conversion.
-<!-- Converting an expression of a given type to another type is known as type conversion. In nGQL, type conversion is divided into implicit conversion and explicit conversion. -->
+Converting an expression of a given type to another type is known as type conversion. 
 
-<!-- ## Implicit type conversion
+## Legacy Compatibility Issues
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/3.data-types/9.type-conversion.md:5]: Legacy compatibility issues

File: docs-2.0/3.ngql-guide/3.data-types/9.type-conversion.md [hunk 4]
```diff
@@ -1,37 +1,75 @@
...
 | toFloat() | Converts an integer or string value to a floating point number. |
 | toInteger() | Converts a floating point or string value to an integer value. |
 | type() | Returns the string representation of the relationship type.  |
--->
\ No newline at end of file
+
+## Examples
+
+```ngql
+nebula> RETURNUNWIND [true, false, 'true', 'false', NULL] AS b RETURN toBoolean(b) AS b
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/3.data-types/9.type-conversion.md:25]: Lacks a space: RETURN UNWIND

**Review by randomJoe211 — COMMENTED** [review]
🥇

File: docs-2.0/3.ngql-guide/3.data-types/4.date-and-time.md [hunk 5]
```diff
@@ -2,7 +2,15 @@
...
-Combined with `YIELD`, functions `date()`, `time()`, `datetime()` all accept empty parameters to return the current date, time and datetime.
+Combined with `RETURN`, functions `date()`, `time()`, `datetime()` all accept empty parameters to return the current date, time and datetime.
+
+## OpenCypher Compatibility
+
+- Parameters `year, month, day, hour, minute, second` are supported. But `nanosecond, timezone` are not supported.
+
+- `localdatetime()`, `duration()` are not supported.
+
+- Most string time formats are not supported. The only exception is `2017-03-04T22:30:40`.
```
[the remark this answers is not in the document]
Comment (whitewum) [reply docs-2.0/3.ngql-guide/3.data-types/4.date-and-time.md:13]: by comparing with openCypher  tck and nebula, it's true.

**Review by randomJoe211 — DISMISSED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**